src/PrestaShopBundle/Resources/views/Admin/Improve/International/TaxRulesGroup/Grid/Blocks/bulk_actions.html.twig

```diff
@@ -0,0 +1,46 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% if grid.actions.bulk|length > 0 and grid.data.records_total > 0 %}
```

Comment:
Same here, why a dedicated bulk actions ? we cannot use default ?

Reply:
I had to copy some logic to embed custom submit button for custom route parameters addition

Reply:
Can you write a twig comment to explain this 😉 ?

Reply:
Instead of twig extension it was redone with custom grid actions. I think it should be a bit cleaner.